xml/cl.xml

```diff
@@ -1835,6 +1835,7 @@ server's OpenCL/api-docs repository.
         <enum value="0x204D"        name="CL_DEVICE_SEMAPHORE_IMPORT_HANDLE_TYPES_KHR"/>
         <enum value="0x204E"        name="CL_DEVICE_SEMAPHORE_EXPORT_HANDLE_TYPES_KHR"/>
         <enum value="0x204F"        name="CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR"/>
+        <enum value="0x????"        name="CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR"/>
```

Comment:
We should assign a value to this enum as part of this PR, probably something from the `0x2052` to `0x2054` range that is currently unused, below.

Reply:
Made the requested changes.

Reply:
Thanks!  Could you update the spec text also?  It still says `0x????`.